app/models/payment_confirmation.rb

```diff
@@ -23,12 +23,22 @@ def ingest
 
       payroll_run.update!(confirmation_report_uploaded_by: admin_user_id)
 
-      errors.empty? || (raise ActiveRecord::Rollback)
+      if errors.empty?
+        payroll_run.claims.each do |claim|
+          ClaimMailer.payment_confirmation(claim, payment_date_timestamp).deliver_later
+        end
+      else
+        raise ActiveRecord::Rollback
+      end
     end
   end
 
   private
 
+  def payment_date_timestamp
+    Date.today.next_occurring(:friday).to_time.to_i
```

Comment:
```suggestion
    Date.today.next_occurring(:friday).to_i
```

Reply:
I don't think `.to_i` is available on Date objects.